src/frontend/komponenter/Behandling/VedtakOgBeregning/VedtaksresultatSwitch.tsx

```diff
@@ -72,6 +73,14 @@ const VedtaksresultatSwitch: React.FC<Props> = (props: Props) => {
         tomVedtaksperiodeRad,
     ]);
 
+    useEffect(() => {
+        if (lagretVedtak) {
+            settPeriodeBegrunnelse(lagretVedtak.periodeBegrunnelse);
+            settInntektBegrunnelse(lagretVedtak.inntektBegrunnelse);
+            settVedtaksperiodeListe(lagretVedtak.perioder);
+        }
+    }, []);
```

Comment:
Lurer på om du trenger en `}, [NOE]);` her?  @alexandraback

Reply:
`lagretVedtak` er ju en klar dependency her og borde dermed inkluderas i dependency arrayen.  I teorin ska eslint ge ett felmeddelande av typen `Either include it or remove the dependency array react-hooks/exhaustive-deps` i slike tilfellen som detta

Reply:
Done ✅